packages/trust/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@web3-onboard/trust",
+  "version": "2.0.0",
```

```suggestion
  "version": "2.0.0-alpha.1",
```
For staging testing purposes prior to public package release